src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldMedia.class.php

```diff
@@ -58,12 +58,13 @@ protected function getHtmlRenderedHtml($bReadOnly)
         parent::GetHTML();
         $aImageData = array();
         $iPosition = 0;
+        $this->oTableConf = $this->oTableRow->GetTableConf();
+        
         /* @var $oImage TCMSImage */
         while ($oImage = $oImages->Next()) {
             $oViewRenderer = $this->getViewRenderer();
             $oViewRenderer->addMapperFromIdentifier('chameleon_system_core.mapper.media_field_image_box');
             $oViewRenderer->AddMapper(new TPkgCmsTextfieldImage());
-            $this->oTableConf = $this->oTableRow->GetTableConf();
```

Comment:
That is not necessary anymore?

Reply:
I moved this line above the loop, because it's not necessary to do this for every image.